Nautilus/Patchers/CustomSoundPatcher.cs

```diff
@@ -923,4 +950,18 @@ internal static void SetChannel3DAttributes(Channel channel, Vector3 position)
         ATTRIBUTES_3D attributes = position.To3DAttributes();
         channel.set3DAttributes(ref attributes.position, ref attributes.velocity);
     }
+    
+    private static bool TryFadeOutBeforeStop(Channel channel)
+    {
+        if (channel.getCurrentSound(out var sound) != RESULT.OK || !FadeOuts.TryGetValue(sound.handle, out var fadeOut))
+        {
+            channel.stop();
+            return false;
+        }
+            
```

Comment:
```suggestion

channel.getDelay(out ulong _, out ulong _, out bool stopChannels);
        
if (stopChannels)
    return false;
    
```

This prevents unnecessary and conflicting fade points from cluttering the sound channel.